stories/list/object/object-list.stories.js

```diff
@@ -0,0 +1,1003 @@
+import '../../../dist/list.css';
+import '../../../dist/icon.css';
+import '../../../dist/checkbox.css';
+
+export default {
+    title: 'Components/List/Object',
```

Comment:
@jessmetivier Should this be under "List" or grouped with the other "Object" components. This looks a little weird to me:

![Screen Shot 2020-09-25 at 9 59 52 AM](https://user-images.githubusercontent.com/29607818/94295303-e33c5700-ff15-11ea-9044-7994eaa92f92.png)

Reply:
As per Fiori3 design guidelines, the object list item is one of the list variants and it will be reusing most the list and standard list item styles. 
Following the above points, it is placed under the list. Please let me know if that is not correct understanding, so I can place it accordingly.